File: scripts/write-file.js
const fs = require('fs');
const path = require('path');
const fp = process.argv[2];
const c = process.argv[3];
if (!fp || !c) { console.error("no args"); process.exit(1); }
const d = path.dirname(fp);
if (!fs.existsSync(d)) fs.mkdirSync(d, { recursive: true });
fs.writeFileSync(fp, c);
console.log("OK:" + fp.length);

File: scripts/write-file.py
import base64, sys, os, json

def write_file(path, content):
    os.makedirs(os.dirname(path), exist_ok=True)
    with open(path, 'w') as f:
        f.write(content)
    print(f'Written: {path}')

if __name__ == '__main__':
    path = sys.argv[1]
    encoded = sys.argv[2]
    content = base64.b64decode(encoded).decode('utf-8')
    write_file(path, content)